.circleci/config.yml

```diff
@@ -390,13 +380,41 @@ commands:
             git status
             gh pr create --fill --base main --head $CIRCLE_BRANCH
 
+  build-and-push-custom-node-image:
+    parameters:
+      image-name:
+        type: string
+    steps:
+      - run:
+          name: Build  Docker Image
+          command: docker build -f BuilderImage.Dockerfile . -t ${AWS_ACCOUNT_ID_DEV}.dkr.ecr.${AWS_REGION}.amazonaws.com/navigator_builder:<< parameters.image-name >>-${CIRCLE_SHA1} -t ${AWS_ACCOUNT_ID_DEV}.dkr.ecr.${AWS_REGION}.amazonaws.com/navigator_builder:<< parameters.image-name >> -t ${AWS_ACCOUNT_ID_DEV}.dkr.ecr.${AWS_REGION}.amazonaws.com/navigator_builder:<< parameters.image-name >>-latest
+      - run:
+          name: Push Builder Docker Image
+          command: |
+            aws ecr get-login-password --region ${AWS_REGION} | docker login --username AWS --password-stdin ${AWS_ACCOUNT_ID_DEV}.dkr.ecr.${AWS_REGION}.amazonaws.com
+            docker push ${AWS_ACCOUNT_ID_DEV}.dkr.ecr.${AWS_REGION}.amazonaws.com/navigator_builder:<< parameters.image-name >>-${CIRCLE_SHA1}
+            docker push ${AWS_ACCOUNT_ID_DEV}.dkr.ecr.${AWS_REGION}.amazonaws.com/navigator_builder:<< parameters.image-name >>
+            docker push ${AWS_ACCOUNT_ID_DEV}.dkr.ecr.${AWS_REGION}.amazonaws.com/navigator_builder:<< parameters.image-name >>-latest
+
+  docker-setup-node-image:
+    steps:
+      - install-awscli
+      - setup_remote_docker:
+          version: 20.10.14
+          docker_layer_caching: true
+      - checkout
+      - build-and-push-custom-node-image:
+          image-name: node_v16_16
```

Comment:
This image name is still confusing. While this image is based on node, its not what the image is. The image is a builder for the BFS navigator.

Reply:
Changed the name to bfs_navigator_builder.